server/src/main/resources/application.yaml

```diff
@@ -2,4 +2,7 @@ spring:
   jpa:
     hibernate:
       ddl-auto: update
-    show-sql: true
\ No newline at end of file
+    show-sql: true
+
+jwt:
+  signerKey: "dzHLIcCikdrF5qIn3uBIKiZ6ZSMmjB0b89cvR1+cIBynshiluVVrWpFr5v3he9C4"
```

Comment:
Don't add private key in code

Reply:
Take a look [here](https://medium.com/@Marou_arnault/spring-boot-hide-your-credentials-fb1ca22ae911)